aviary/examples/run_aviary_example.py

```diff
@@ -87,7 +87,6 @@
     },
     "post_mission": {
         "include_landing": False,
-        "constrain_range": True,
```

Comment:
I see.  You got rid of "constrain_range" because it is redundant to have it and target_range. If you specify a target range, then you obviously are constraining it.